src/js/routes/services.js

```diff
@@ -13,7 +13,7 @@ let serviceRoutes = {
   children: [
     {
       type: Route,
-      name: 'services-deployments',
+      name: '-services-deployments',
```

Comment:
Route names are deprecated since react-router 1.0, so I wouldn't worry too much about it.